Dockerfile-7.1

```diff
@@ -221,6 +225,7 @@ RUN rm -f \
     mv /etc/php/${PHP_VER}/mods-available/radius.ini         /etc/php/extra-mods/radius/ &&\
     mv /etc/php/${PHP_VER}/mods-available/raphf.ini          /etc/php/extra-mods/raphf/ &&\
     mv /etc/php/${PHP_VER}/mods-available/redis.ini          /etc/php/extra-mods/redis/ &&\
+    mv /etc/php/${PHP_VER}/mods-available/soap.ini         /etc/php/extra-mods/soap/ &&\
```

Comment:
Just saw I missed file indentation here and below (L 255), just in this file. Sorry for that.
Do you me to push a new PR?